BridgeAppSDK/SBARoundedButton.swift

```diff
@@ -79,7 +80,16 @@ public let SBARoundedButtonDefaultCornerRadius: CGFloat = SBARoundedButtonDefaul
     }
     
     public required init() {
-        super.init(frame: CGRect(x: 0, y: 0, width: 250, height: 51))
+        super.init(frame: CGRect(x: 0, y: 0, width: SBARoundedButtonDefaultWidthWith1Button, height: SBARoundedButtonDefaultHeight))
+        commonConstraintSetup(1)
+    }
+    
+    public init(numberOfButtons: Int) {
+        super.init(frame: CGRect(x: 0, y: 0, width: SBARoundedButtonDefaultWidthWith2Buttons, height: SBARoundedButtonDefaultHeight))
```

Comment:
no, this is a work-around to Josh's stuff and will be updated using constraints.